.github/workflows/test.yml

```diff
@@ -0,0 +1,75 @@
+name: Test
+on: push
+
+
+concurrency:
+  group: ci-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  provision:
+    runs-on: ubuntu-latest
+    container:
+      image: clojure:openjdk-11-lein-2.9.6
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Cache deps
+      uses: actions/cache@v2
+      with:
+        path: /root/.m2
+        key: v1-deps-${{ hashFiles('project.clj') }}
+        restore-keys: |
+          v1-deps-${{ hashFiles('project.clj') }}
+
+    - name: Install dependencies
+      run: |
+        lein deps
+
+  test:
+    name: test with ${{ matrix.version }}
+    needs: provision
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "postgres:latest"
+          - "postgres:13"
```

Comment:
As discussed - for Lockjaw we can drop matrix builds, since it doesn't depend on a particular version of PG anymore (it used to require 11+ but that's pretty much standard nowadays)

Reply:
13 is fine 👍